app/backend/app.py

```diff
@@ -43,6 +45,10 @@
 async def index():
     return await bp.send_static_file("index.html")
 
+@bp.route("/redirect")
+async def redirect():
```

Comment:
comment - empty page required for redirect to work correctly.

Reply:
I'm intrigued as to why this is empty, as I've typically had the redirect go to somewhere in previous samples, like 
https://github.com/Azure-Samples/ms-identity-python-webapp/blob/14a6be5ad1deeb38d89707e260620c0d5a4c9494/app.py#L38

How come it's different in this case? Does the user not land up on the /redirect themselves?